doc/build_wamr.md

```diff
@@ -114,6 +114,19 @@ Currently we only profile the memory consumption of module, module_instance and
 
 > The function name searching sequence is the same with dump call stack feature.
 
+#### **Enable the global heap**
+- **WAMR_BUILD_GLOBAL_HEAP_POOL**=1/0, default to disable if not set for all *iwasm* applications, except for the platforms Alios and Zephyr.
+
+> **WAMR_BUILD_GLOBAL_HEAP_POOL** is used in the *iwasm* applications provided in the directory `product-mini`. When writing your own host application using WAMR, you must set the initialization argument `mem_alloc_type` to allocate and use a global heap.
```

Comment:
A little confusing about `you must set the initialization argument `mem_alloc_type` to allocate and use a global heap`, will it make developer think that he **must** set argument to `Alloc_With_Pool`?
How about changing to something like:
```md
When writing your own host application using WAMR, if you want to use a global heap and allocate memory from it, you must set the initialization argument `mem_alloc_type` to `Alloc_With_Pool`.
```

Reply:
Much better!